README.md

```diff
@@ -93,3 +93,10 @@ These images are no longer being published:
 You can follow the detailed guide on how to contribute [here](CONTRIBUTING.md).
 
 [![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/#https://github.com/gitpod-io/workspace-images)
+
+## Where to follow new updates
+
+- **Announcements**: Significant changes and breaking updates are announced through our [Changelog](https://www.gitpod.io/changelog/) with details.
```

Comment:
Can you please change this to link to our `CHANGELOG.md`? We decided to post updates there instead.

Reply:
Thanks! Didn't know.